pkg/autoscaler/autoscaler/util_test.go

```diff
@@ -110,87 +110,359 @@ func TestCheckStsAutoScalingInterval(t *testing.T) {
 func TestDefaultTac(t *testing.T) {
 	g := NewGomegaWithT(t)
 	tc := newTidbCluster()
-	tac := newTidbClusterAutoScaler()
-	tac.Spec.TiDB = nil
-	tac.Spec.TiKV.ScaleOutIntervalSeconds = nil
-	tac.Spec.TiKV.ScaleInIntervalSeconds = nil
-	tac.Spec.TiKV.Rules = map[corev1.ResourceName]v1alpha1.AutoRule{
-		corev1.ResourceCPU: {
-			MaxThreshold: 0.8,
+	testcases := []struct {
+		name      string
+		sourceTac *v1alpha1.TidbClusterAutoScaler
+		expectTac *v1alpha1.TidbClusterAutoScaler
+	}{
+		// case1
```

Comment:
This case can be covered by case 6, we just need to keep one of them

Reply:
updated.